src/foam/dao/HTTPSink.java

```diff
@@ -76,6 +76,7 @@ public void put(Object obj, Detachable sub) {
           propertyPredicate_ == null ?
           new foam.lib.json.Outputter(getX()).setOutputDefaultValues(outputDefaultValues_).setPropertyPredicate(new NetworkPropertyPredicate()) :
           new foam.lib.json.Outputter(getX()).setOutputDefaultValues(outputDefaultValues_).setPropertyPredicate(propertyPredicate_);
+        ((foam.lib.json.Outputter) outputter).setOutputReadableDates(false);
```

Comment:
Is this going to break any existing integrations?

Reply:
There is no existing integration using JSON DugRule yet.